x-pack/plugins/cases/public/components/user_actions/index.test.tsx

```diff
@@ -452,18 +356,160 @@ describe(`UserActions`, () => {
       const props = {
         ...defaultProps,
         userProfiles: userProfilesMap,
-        caseUserActions: isolateAction,
         data: {
           ...defaultProps.data,
           comments: [hostIsolationComment({ createdBy: { profileUid: userProfiles[0].uid } })],
         },
       };
 
+      useFindCaseUserActionsMock.mockReturnValue({
+        ...defaultUseFindCaseUserActions,
+        data: { userActions: isolateAction },
+      });
+
       appMockRender.render(<UserActions {...props} />);
 
       expect(screen.getByTestId('case-user-profile-avatar-damaged_raccoon')).toBeInTheDocument();
       expect(screen.getByText('DR')).toBeInTheDocument();
       expect(screen.getByText('Damaged Raccoon')).toBeInTheDocument();
     });
   });
+
+  describe('pagination', () => {
```

Comment:
Could you please add a test where we show 20 items (the show more button is not shown) then we add another user action and then check if the show more button shows and the number of total user actions shown is correct?